src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php

```diff
@@ -52,8 +52,12 @@ public function findLatestCartByChannelAndCustomer(ChannelInterface $channel, Cu
 
     public function getTotalSalesForChannel(ChannelInterface $channel): int;
 
+    public function getTotalPaidSalesForChannel(ChannelInterface $channel): int;
+
     public function countFulfilledByChannel(ChannelInterface $channel): int;
 
+    public function countPaidByChannel(ChannelInterface $channel): int;
+
     /**
```

Comment:
Wrong order

Reply:
in interfaces, we don't have any rules for order of functions so I don't know what order will be fine :D

Reply:
The order of methods should be... rational :D It's seems good IMO